tests/cache/models.py

```diff
@@ -11,3 +11,7 @@ class Poll(models.Model):
     question = models.CharField(max_length=200)
     answer = models.CharField(max_length=200)
     pub_date = models.DateTimeField('date published', default=expensive_calculation)
+
+
+class UnpickableType(object):
```

Comment:
I guess the class has to live at the module level and couldn't be included inline in the test? Probably would be fine to include it in `tests.py` since it's not a model.